src/backEnd/utils/calendarOperations.js

```diff
@@ -1,49 +0,0 @@
-const google = require('googleapis');
-const calendar = google.calendar('v3');
-
-const deleteEvent =(auth,calendarId,eventId,cb)=> {
-  calendar.events.delete({
```

Comment:
calendar.events is used multiple times in this file. consider just making it a variable.